File: app/src/main/java/com/blog/demo/LogTool.java
package com.blog.demo;

import android.util.Log;

public class LogTool {

    public static void logi(String tag, String message) {
        Log.i("com.blog.demo/" + tag, message);
    }

    public static void loge(String tag, Throwable throwable) {
        Log.e("com.blog.demo/" + tag, "", throwable);
    }

}

File: app/src/main/java/com/blog/demo/LogTool.kt
package com.blog.demo

import android.util.Log

object LogTool {

    @JvmStatic
    fun logi(tag: String, message: String?) {
        Log.i("com.blog.demo/$tag", message ?: "")
    }

    @JvmStatic
    fun loge(tag: String, throwable: Throwable?) {
        Log.e("com.blog.demo/$tag", "", throwable)
    }

}